robottelo/helpers.py

```diff
@@ -532,7 +533,17 @@ def create_repo(name, repo_fetch_url=None, packages=None, hostname=None):
                     result.stderr,
                     'Unable to download package {}'.format(package),
                 )
-
+    if wipe_repodata:
+        result = ssh.command(
+            'rm -rf {}'.format(os.path.join(repo_path, 'repodata/')),
```

Comment:
os.path.join cannot be used for remote paths

Reply:
That's funny as you were the person who suggested to use os.path.join instead of plain path here 🙂